spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/context/AzureContextAutoConfiguration.java

```diff
@@ -36,7 +36,7 @@
 @Configuration
 @EnableConfigurationProperties(AzureProperties.class)
 @ConditionalOnClass(Azure.class)
-@ConditionalOnProperty(value = "spring.cloud.azure.enabled", matchIfMissing = true)
+@ConditionalOnProperty(value = "spring.cloud.azure.credentialFilePath")
```

Comment:
we will refine property name to some thing like `credential-file-path`, right ?

Reply:
Yes, in another PR.